demos/python_demos/facies_classification_demo/README.md

```diff
@@ -0,0 +1,46 @@
+# Facies classification Python* Demo
+
+This demo demonstrate how to run facies classification using OpenVINO&trade;
+
+
+## How It Works
+Upon the start-up, the demo application loads a network and an given dataset file to the Inference Engine plugin. When inference is done, the application displays 3d itkwidget viewer with facies interpretation.
+
+## Running
+
+### Installation of dependencies
+
+Steps to create CPU extension:
+
+```bash
+source /opt/intel/openvino/bin/setupvars.sh
+export TBB_DIR=/opt/intel/openvino/deployment_tools/inference_engine/external/tbb/cmake/
+
+cd user_ie_extensions
```

Comment:
Don't forget to add extensions code to your PR

Reply:
there already are. -  demos/python_demos/facies_classification_demo/user_ie_extensions/